services/app-web/src/pages/SubmissionSummary/V2/SubmissionSummaryV2.tsx

```diff
@@ -0,0 +1,299 @@
+import { NavLink } from 'react-router-dom'
+import { useAuth } from '../../../contexts/AuthContext'
+import { packageName } from '../../../common-code/healthPlanFormDataType'
+import { ContractDetailsSummarySectionV2 } from '../../StateSubmission/ReviewSubmit/V2/ReviewSubmit/ContractDetailsSummarySectionV2'
+import { ContactsSummarySection } from '../../StateSubmission/ReviewSubmit/V2/ReviewSubmit/ContactsSummarySectionV2'
+import { RateDetailsSummarySectionV2 } from '../../StateSubmission/ReviewSubmit/V2/ReviewSubmit/RateDetailsSummarySectionV2'
+import { SubmissionTypeSummarySectionV2 } from '../../StateSubmission/ReviewSubmit/V2/ReviewSubmit/SubmissionTypeSummarySectionV2'
+import {
+    SubmissionUnlockedBanner,
+    SubmissionUpdatedBanner,
+    DocumentWarningBanner,
+} from '../../../components'
+import { Loading } from '../../../components'
+import { usePage } from '../../../contexts/PageContext'
+import {
+    useFetchContractQuery,
+    UpdateInformation,
+} from '../../../gen/gqlClient'
+import { ErrorForbiddenPage } from '../../Errors/ErrorForbiddenPage'
+import { Error404 } from '../../Errors/Error404Page'
+import { GenericErrorPage } from '../../Errors/GenericErrorPage'
+import styles from '../SubmissionSummary.module.scss'
+import { ChangeHistoryV2 } from '../../../components/ChangeHistory/ChangeHistoryV2'
+import { UnlockSubmitModalV2 } from '../../../components/Modal/V2/UnlockSubmitModalV2'
+import { useLDClient } from 'launchdarkly-react-client-sdk'
+import { featureFlags } from '../../../common-code/featureFlags'
+import { RoutesRecord } from '../../../constants'
+import { useRouteParams } from '../../../hooks'
+
+function UnlockModalButton({
+    disabled,
+    modalRef,
+}: {
+    disabled: boolean
+    modalRef: React.RefObject<ModalRef>
+}) {
+    return (
+        <ModalToggleButton
+            modalRef={modalRef}
+            className={styles.submitButton}
+            data-testid="form-submit"
+            disabled={disabled}
+            outline
+            opener
+        >
+            Unlock submission
+        </ModalToggleButton>
+    )
+}
+
+export const SubmissionSummaryV2 = (): React.ReactElement => {
+    // Page level state
+    const { updateHeading } = usePage()
+    const modalRef = useRef<ModalRef>(null)
+    const [pkgName, setPkgName] = useState<string | undefined>(undefined)
+    const [documentError, setDocumentError] = useState(false)
+    const { loggedInUser } = useAuth()
+
+    useEffect(() => {
+        updateHeading({ customHeading: pkgName })
+    }, [pkgName, updateHeading])
+    const { id } = useRouteParams()
+
+    const ldClient = useLDClient()
+    const showQuestionResponse = ldClient?.variation(
+        featureFlags.CMS_QUESTIONS.flag,
+        featureFlags.CMS_QUESTIONS.defaultValue
+    )
+
+    // API requests
+    const {
+        data: fetchContractData,
+        loading: fetchContractLoading,
+        error: fetchContractError,
+    } = useFetchContractQuery({
```

Comment:
I know Hana isn't sold on it but I still think that the ApolloQueryWrapper is useful, makes some impossible cases impossible